src/core/StelObject.cpp

```diff
@@ -677,6 +676,67 @@ QString StelObject::getCommonInfoString(const StelCore *core, const InfoStringGr
 			res += q_("Polar dawn") + "<br />";
 		else if (rts[0]<99.f && rts[2]>99.f)
 			res += q_("Polar dusk") + "<br />";
+
+		// Greatest Digression: limiting azimuth and hour angles for circumpolar stars
+		// This may fill 2 lines where RTS just says "circumpolar"
+		double dec_equ, ra_equ;
+		StelUtils::rectToSphe(&ra_equ,&dec_equ,getEquinoxEquatorialPos(core));
+		const double latitude=static_cast<double>(core->getCurrentLocation().latitude)*M_PI_180;
+		if (((latitude>0.) && (dec_equ>=latitude)) || ((latitude<0.) && (dec_equ<=latitude)))
```

Comment:
Why? On my latitude (53 degrees) I'll see objects as circumpolar with declination since 37 degrees and up to 90 degrees. In your code I'll see objects as circumpolar with declination since 53 degrees, but this is wrong.

Reply:
The code is OK, but the documentation is faulty. This only is for objects culminating between pole and zenith.

Reply:
OK, please fix the docs and merge the code :)